dali/kernels/imgproc/color_manipulation/brightness_contrast.h

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_IMGPROC_COLOR_MANIPULATION_BRIGHTNESS_CONTRAST_H_
+#define DALI_KERNELS_IMGPROC_COLOR_MANIPULATION_BRIGHTNESS_CONTRAST_H_
+
+#include "dali/util/ocv.h"
+#include "dali/core/common.h"
+#include "dali/core/convert.h"
+#include "dali/core/geom/box.h"
+#include "dali/core/error_handling.h"
+#include "dali/kernels/kernel.h"
+#include "dali/pipeline/data/types.h"
+
+namespace dali {
+namespace kernels {
+
+namespace brightness_contrast {
+
+template<size_t ndims, class Roi>
+TensorShape<ndims> to_shape(Roi roi, size_t nchannels) {
```

Comment:
```suggestion
template <size_t roi_dims, typename RoiCoord
TensorShape<roi_dims+1> to_shape(Box<roi_dims, RoiCoord> roi, size_t nchannels) {
```
This will allow for deduction of shape dims from box dims.